purchase_security/models/__init__.py

```diff
@@ -1,2 +1,7 @@
-from . import purchase_order
+from . import ir_rule
 from . import purchase_team
+from . import res_partner
+from . import res_users
+
+# Set at the end to avoid access error to the res.users field.
+from . import purchase_order
```

Comment:
What is the error? Reproducible here on runboat? Don't let anyway any empty line